tests/fixtures/couplet.json

```diff
@@ -1 +1,208 @@
\ No newline at end of file
+[
+  {
+    "tokens": [
+      {
+        "word": [
+          {
+            "syllable": "Aun",
+            "is_stressed": true
+          },
+          {
+            "syllable": "que",
+            "is_stressed": false
+          }
+        ],
+        "stress_position": -2
+      },
+      {
+        "word": [
+          {
+            "syllable": "la",
+            "is_stressed": false
+          }
+        ],
+        "stress_position": 0
+      },
+      {
+        "word": [
+          {
+            "syllable": "mo",
+            "is_stressed": true
+          },
+          {
+            "syllable": "na",
+            "is_stressed": false
+          }
+        ],
+        "stress_position": -2
+      },
+      {
+        "word": [
+          {
+            "syllable": "se",
+            "is_stressed": false
+          }
+        ],
+        "stress_position": 0
+      },
+      {
+        "word": [
+          {
+            "syllable": "vis",
+            "is_stressed": true
+          },
+          {
+            "syllable": "ta",
+            "is_stressed": false
+          }
+        ],
+        "stress_position": -2
+      },
+      {
+        "word": [
+          {
+            "syllable": "de",
+            "is_stressed": false
+          }
+        ],
+        "stress_position": 0
+      },
+      {
+        "word": [
+          {
+            "syllable": "se",
+            "is_stressed": true
+          },
+          {
+            "syllable": "da",
+            "is_stressed": false
+          }
+        ],
+        "stress_position": -2
+      },
+      {
+        "symbol": ","
+      }
+    ],
+    "phonological_groups": [
+      {
+        "syllable": "Aun",
+        "is_stressed": true
+      },
+      {
+        "syllable": "que",
+        "is_stressed": false
+      },
+      {
+        "syllable": "la",
+        "is_stressed": false
+      },
+      {
+        "syllable": "mo",
+        "is_stressed": true
+      },
+      {
+        "syllable": "na",
+        "is_stressed": false
+      },
+      {
+        "syllable": "se",
+        "is_stressed": false
+      },
+      {
+        "syllable": "vis",
+        "is_stressed": true
+      },
+      {
+        "syllable": "ta",
+        "is_stressed": false
+      },
+      {
+        "syllable": "de",
+        "is_stressed": false
+      },
+      {
+        "syllable": "se",
+        "is_stressed": true
+      },
+      {
+        "syllable": "da",
+        "is_stressed": false
+      }
+    ],
+    "rhythm": {
+      "rhythmical_stress": "+--+--+--+-",
+      "type": "pattern"
+    },
+    "rhythmical_length": 11
```

Comment:
This is the all format of `rhythm` dict, isn't?

Reply:
Those fixtures need some more work actually. All of them have the enconding problem of converting to json, will solve it

Reply:
I fixed by using the one-liner JSON and writing directly to a file, not to a text output like Notebook.